### PR TITLE
Feature embedded skin visualization process

### PR DESCRIPTION
**Pull request by rubenzorrilla**

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp [hunk 1]
```diff
@@ -0,0 +1,590 @@
...
+    Parameters default_parameters( R"(
+    {
+        "shape_functions"                     : "standard",
+        "reform_model_part_at_each_time_step" : false,
+        "visualization_variables"             : ["VELOCITY","PRESSURE","DISTANCE"]
+    })");
+
+    rParameters.ValidateAndAssignDefaults(default_parameters);
+
+    mShapeFunctions = rParameters["shape_functions"].GetString();
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp]: you should check if the shape_functions name is in the list of admissible options

File: applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp [hunk 2]
```diff
@@ -0,0 +1,590 @@
...
+        case GeometryData::KratosGeometryType::Kratos_Line2D2:
+            return Kratos::make_shared<Line2D2< Node<3> > >(rNewNodesArray);
+        case GeometryData::KratosGeometryType::Kratos_Triangle3D3:
+            return Kratos::make_shared<Triangle3D3< Node<3> > >(rNewNodesArray);
+        default:
+            KRATOS_ERROR << "Implement the visualization for the intersection geometry type " << rOriginGeometryType;
+    }
+}
+
+};  // namespace Kratos.
```
Comment (RiccardoRossi) [inline applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp]: add the newline ;-)

**Review by RiccardoRossi — DISMISSED** [review]
minor comments

File: applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp (hunk 2 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp]: Done!

**Review by jcotela — APPROVED** [review]

**Pull request merged**